.github/CONTRIBUTING.md

```diff
@@ -169,12 +169,32 @@ See [Contributing to documentation related to PowerShell](#contributing-to-docum
 
 #### Pull Request - Code Review
 
-* After a successful test pass,
-  the area maintainers will do a code review,
+* Roles of a PR: Author, Reviewer, and Assignee
+  * Reviewer and Assignee are two separate roles of a PR.
+    * A Reviewer can be anyone from the collaborators.
+      A Reviewer reviews the change of a PR,
+      leaves comments for the Author to address,
+      and approves the PR when the change looks good.
+    * An Assignee must be a maintainer, who monitors the progress of the PR,
+      coordinates the review process, and merges the PR after it's approved.
+      The Assignee may or may not be a Reviewer of the PR at the same time.
+  * An Author is encouraged to choose Reviewer(s) and Assignee for the PR.
+    If no Assignee is chosen, one of maintainers shall be assigned to it.
+    If no Reviewer is chosen, Assignee shall choose Reviewer(s) as appropriate.
+    * If an Author is a [PowerShell Team](https://github.com/orgs/PowerShell/people) member,
+      then the Author **is required** to choose Reviewer(s) and Assignee of the PR.
+  * The Assignee may be re-assigned by maintainers,
+    and more Reviewer(s) may be added by Assignee as appropriate.
+
+* An Author **is responsible** for driving the PR to the Approved state.
+  After a successful test pass, Reviewer(s) can start a code review,
   commenting on any changes that might need to be made.
+  The Author addresses the comments, and pings Reviewer(s) to start the next iteration.
+  If the review is making no progress (or very slow),
+  the Author can always ask the Assignee to help coordinate the process and keep it moving.
 
```

Comment:
I think labels are more useful to maintainers and less useful to reviewers because a reviewer probably will not look at the PR list very often. So I think it's better that the author pings the reviewers when she/he is ready to start the next review iteration.